pkg/dr/health.go

```diff
@@ -0,0 +1,81 @@
+package dr
+
+import (
+	ctx "context"
+	"encoding/base64"
+	"encoding/json"
+	"fmt"
+	"strings"
+
+	log "github.com/sirupsen/logrus"
+
+	"github.com/rook/kubectl-rook-ceph/pkg/exec"
+	"github.com/rook/kubectl-rook-ceph/pkg/k8sutil"
+	rookv1 "github.com/rook/rook/pkg/apis/ceph.rook.io/v1"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+type secretData struct {
+	Key     string `json:"key"`
+	MonHost string `json:"mon_host"`
+}
+
+func Health(context *k8sutil.Context, operatorNamespace, cephClusterNamespace string, args []string, ExitOnError bool) {
+	fmt.Println("INFO: fetching the cephblockpools with mirroring enabled")
+	blockPoolList, err := context.RookClientset.CephV1().CephBlockPools(cephClusterNamespace).List(ctx.TODO(), v1.ListOptions{})
+	if err != nil {
+		log.Error(err)
+	}
+
+	var mirrorBlockPool rookv1.CephBlockPool
+	for _, blockPool := range blockPoolList.Items {
+		if blockPool.Spec.Mirroring.Enabled {
+			mirrorBlockPool = blockPool
+			fmt.Printf("found %s cephblockpool with mirroring enabled\n", mirrorBlockPool.Name)
```

Comment:
Quotes around the resource name is a nice pattern, even if it is slightly different from the script plugin
```suggestion
			fmt.Printf("found %q cephblockpool with mirroring enabled\n", mirrorBlockPool.Name)
```

Reply:
got it